### DISCUSSION TITLE
WIP:Psionics, GoI Interaction, GOC Psionics Division.

### DISCUSSION THREAD

**Discussion by ghost** [Foundation-19/Foundation-19#916] [Design]
Psionics are powers originating from extremely uncommon genetic mutations subtly present in but not limited to human species, most of the time they are not activated and a man can live his whole life not aware of his own capabilities unless triggered by mental trauma or chem induced awakening. It's a known fact that some of GoI/SCP's show interests in nurturing or manipulating these inherent powers, some hunt them down.

We are currently using the Bay psionics.
https://bay.ss13.me/en/Guides/Psionics

## Abstract
Psionics in the game as they are an immediate mark of an antagonist and probably should be promptly executed foregoing arrest because of their uncontrollable nature, there's barely any chance for a regular man to gain psionics and CURRENTLY, for a good reason because there's not much point to power up any staff, exceptionally with how easy it actually is to gain higher ranks of psionic power if you just know few things from scouring code or wikis.

I think we should include psionics in the game with a bit more thought because they do exist in SCP lore and deserve to be highlighted with some respect.

### Goals
- Make Psionics more common.
- Create a device(collars,reality-anchor contact-grenades) capable of suppressing psionics(?made from nullglass?).
- Make Psionics visually detectable at Master rank.
- Limit the chem induceable psionic ranks to only be able to reach Master rank.
- Create a stationary device/machinery capable of exceeding Master rank to Grandmaster.
- Paramounts never occur naturally and if they are spawned, they should be more powerfull.
- Create a GOC Psionics lab off-site
- Introduce more reasonable Psionic-GoI's to D class interactions.
- Make it OOC known that any non-government psychics are generally handed over to different GoI's in order to train/discipline for a price, of course.
- Make Psychic D-Class more interesting.
- Have one huge luxury enclosure for D-Class Psychics waiting for transfer.

### Non-goals
- Don't ever make FBPs Psychics.

### Up the percentage of possible psionics.
- **To 20% for regular staff members.**
To warrant research staff going out of their set way to seek out possible psychics within staff because D class pop is not guaranteed and brings in flavor to the round.

- **To 20% for D classes, with 5% starting as Operant in one tree.**

### Foundation Interaction with Psionics.
Foundation at it's core would either suppress and isolate psionics or trade them out with GOC in turn for some nice things, SCP's are their main obsession afterall and their approach to subjects/employees is way less attached, almost sterile than that of GOC would have.
Ethics and medical/research staff and psychiatrists have to step in to oversee handling of psionics to make sure they don't go insane(or admin acts on staff negligence) in reeducation with routine trainings and employing use of psychic collars/amnestics.

### GoI Interaction with Psionics.
Unless the presence of Psychics in Foundation is warranted(read type Greens) for indefinite containment then it can be fair to leave them in their hands but any undue treatment of them can be a point of tension between SCPF and other GoI's. Whether it's to break them out, recruit them, kill them, post-mortem revenge or otherwise.

GOC should focus on the human development of psionics and as such should be supplied with devices capable of exceeding Master-rank psionics for their operatives off-site in addition, the forefront of their forces should be consisting of the most exemplary, disciplined and stable operators that were given proper treatment to excel with potential recruitment of psionic D class personnel for favors could be interesting part of both tension and reward.

. . .
SH - Smuggling psionics out of Foundation for their own ends, Recruitment in addition to their operatives highly possibly being psionics themselves, maybe also capable of rising above Master naturally(Could be capable of buying the stationary psionic rank-rising device but a hacked version not requiring Psychic's Director's ID and one able of rising one to a Paramount rank. VERY EXPENSIVE.)

CI - Vying for the psychics to hate the foundation and aid them in destroying it.

Three Moons Initiative - Possible diplomatic recruitment?

UIU - Probably would just let other organisations deal with it or atleast pass the note down to GOC.

#### GOC Psychic Division Site Staff.
1. Psychic Division Director(This role is too powerfull so probably a whitelist,for the love of god be strict with this role), mostly the same as Consultant/Mentalist with Grandmaster in all trees, maybe Paramount in Redaction. Higher rank. Allowed to create agreements with the Foundation on behalf of the GOC Psychic Division. Shouldn't be the one going as an envoy to the Foundation, meant to be saved for when things get really bad, like an XK scenario or the whole team gets murdered and he can pick up their pieces.</br></br>
2. Psychic Consultant/Mentalist, knowledgable in handling, disciplining and training psionic personnel, oversight of psionic treatments and acts as envoy but can push this duty to the most senior operative, serves under the Director. This class should start with:
- Grandmaster Redaction.
- Master Coercion
- Master Psychokinesis.
3. Psychic Division Operatives, ready to embark on missions either regarding escort and surface abnormalities or rogue psionics, mostly being psychically reviewed by the Consultant/Mentalist, under constant training physically and mentally or furthering their hobbies with fellow operatives.**Timelocked for Recruits by like, 30 hours).**</br>
3.1.1. Tinkerers. - Master Psychokinesis - Operant Energetics - Latent Coercion
3.1.2. Disruptors. - Master Energetics - Operant Coercion - Latent Psychokinesis
3.1.3. Fleshmenders - Master Redaction - Operant Psychokinesis - Operant Coercion

4. Recruits, basically Unproven Operatives needing to build rapport with their handlers and to learn their abilities on how they would cooperate as a team, random skill trees to encourage learning.</br>
4.1. Psionic Prodigy Recruit. - 1 random operant and 2 latent level.
4.2. Psionic Specialized Recruit. - 1 random master and 1 latent level.

#### Protocol regarding psionics.

1. Hostile paramounts are to be immediately isolated and pacified(not killed) with the use of reality-anchor contact-grenades that render psychic powers unusable or an MTF force if necessary and then contained with null collars and straitjackets OR in case of a peacefull Paramount, handed over to the Psionics Division if they won't be undergoing testing for a pretty credit sum(or something).</br></br>
2. If the facility notoriously kills or abuses even lowest of psychics roles ranging from GoI Rep, Ethics, SD or if O5 has to act should send for an Investigations team.</br></br>
3. Psychic Division after receiving a valid complaint should prepare a fax detailing the cause of arrival and the nature of the interjection that should be at first transmitted for approval to the Acting Directly along with being sent to every fax machine for general viewing.</br></br>
4. If the Psychic Division has the go ahead by O5 Council but not Locally, an MTF escort should be sent for. Upon arriving at the Foundation a site-wide announcement could also be cool that a "insert-goi" is here to investigate on behalf of SCPF what went wrong and who's responsible by properly interrogating every witness(D-class, scientists, guards).</br></br>
6. If it's deemed that the Foundation mishandles a psychic subjects in a major, unreasonable way or by ignoring them, the Psychic Division should request that the Acting Director hands over rest of the psychic subjects with a small reward, the Psychic Division should assist with this extraction if permitted to.</br></br>
7. In the case that the any staff becomes beligerent with the Psychic Division at any point, it's important to point headstaff over to the complaint fax requesting an investigation and a permission to enact actions they can take by an Acting SD or O5(investigation and acquirement of psychic staff), If this behaviour persists in a way that obstructs proceedings without having MTF on their side, leave the site premises and send for MTF to resolve the conflict(A lot of people's heads can roll for demotion if they fuck this up).</br></br>
8. It is at the most importance that the Investigations team never gets distracted by things happening in the purview like a Breaching Abnormality, D class riot and Lawbreaking unless it's directly in the way of their actions or stated within the agreement. They are an investigation team first and foremost.</br></br>
9. The use of the device capable of rising above Master Rank should be heavily limited to personnel that has proven themselves in the field and as such only the Psychic Division Directors approval should allow employ of the device.</br></br>
10. Any Psychic unit caught going out of their way of training should be isolated and put into a null-collar for varying amounts of time during which they'll commence reeducation until they give their own appeal vowing to not disappoint again.</br></br>
11. The Psychic Division Lab itself simply does not have a summary execution for Psychics unless the threat has a high probability of destroying the lab itself or killing the Psychic Division's Director.</br></br>

### Alternatives
The Foundation could have their own psionic division, but last time I've seen that in lore almost everyone of them died or they were a subject to some experiment?
### Potential Changes
I'd love better names to both indicate the faction belonging AND function for the GOC Psychic Operatives instead of slapping Psychic Division everywhere. "Psychic Division Tinkerer", "Psychic Division Disruptor", "Psychic Division Fleshmender".

Add in psychic instability that very slowly ramps brain damage after a five minute worth of notifications and psychic stabilization pills that overdose at very small amounts causing reversed controls, dizzines, temporarily lowered eye functions. If administered properly, remain active up to 60 minutes.

**Comment by aidan-bowman** [comment]
while i do like having GOC bases in the game (i hope eventually we have a lot of inter-faction gameplay) i don't think i'd like keeping psionics as a permanent feature. it's be better to overhaul them into type-blue crap than to try and integrate them in-game, especially since type blues are already established lore-wise.

**Reply by ghost** [reply]
> while i do like having GOC bases in the game (i hope eventually we have a lot of inter-faction gameplay) i don't think i'd like keeping psionics as a permanent feature. it's be better to overhaul them into type-blue crap than to try and integrate them in-game, especially since type blues are already established lore-wise.

While I would agree with this sentiment if we were in vacuum where no one ever runs Type Green events and antags can't become psionics at a whim etc, I say that they SHOULD have atleast measures that enable indepth interaction with SUCH events/accidents beyond shoot-to-kill. 5% of D class starting at awakened operant level is still VERY rare and mostly weak. What do you mean by not becoming a permament feature, antags can just create/become physics.

**Reply by aidan-bowman** [reply]
I'm saying that psionics should be completely replaced by thaumaturgy to fit with SCP.
